flask_project/campaign_manager/views.py

```diff
@@ -56,37 +65,99 @@
 
 MAX_AREA_SIZE = 320000000
 
+# create session for Database Queries
+Session = sessionmaker(bind=engine)
+Session.configure(bind=engine)
+session = Session()
+session.rollback()
+
+
+@campaign_manager.route('/add_osm_user/<username>', methods=['GET', 'POST'])
```

Comment:
I dont think we need to define a POST method for this route. 
The argument 'username' is passed in the request so a GET method is enough.

Reply:
After all, a GET method doesnt make sense. It should be a POST as it creates an entry in the database.

Reply:
That make sense. I live change that to a post request.